mappings/net/minecraft/world/LunarWorldView.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_5424 net/minecraft/world/LunarWorldView
+	METHOD method_30271 getDayTime ()J
+	METHOD method_30272 getMoonSize ()F
+		COMMENT Gets the current size of the moon in the world.
+		COMMENT
+		COMMENT @return the size of the moon on a scale of {@code 0.0F} to {@code 1.0F}
+	METHOD method_30273 getLunarIndex ()I
```

Comment:
I think we've been calling this moon phase before

Reply:
The texture for the moon's various phases is called `moon_phases`, so we should definitely use that.